frontend/components/SubNav/SubNav.jsx

```diff
@@ -0,0 +1,21 @@
+import { useContext } from 'react';
```

Comment:
Now that you have created this component, I bet you could apply it to the Nav component since navigation links behave similarly.  If you think they are different enough then its all good, but just a thought. That way future changes will apply to both.

Reply:
I'm not 100% sure what you're asking here, do you mind breaking it down a little more?

Reply:
For sure. So inside of both the Nav and the Subnav, you have code for the links themselves that look and behave the same in both. They share the <li> and <Link> elements, and much of their styling is the same as well, except for some added conditional styling in the nav bar links. The subnavigation itself may be a dropdown, but the links within the subnav are similar enough to the links in the Nav that they can likely share a component. 

Notice the similarities between the two -->
Nav:
```
<li className={navStyle.item} key={`${name}${idx}`}>
              <Link
                href={route}
                className={`${navStyle.link} ${
                  route === router ? navStyle.highlighted : undefined
                }`}
              >
                {name}
              </Link>
    {teamList.length !== 0 && name === 'Teams' && <SubNav />}
</li>
```
SubNav:
```
<li className={subNavStyle.item} key={`${team}${idx}`}>
    <Link href={`/teams/${team.id}`}>{team.name}</Link>
</li>
```
You could create a third component called NavLinks (or something that makes more sense to you), and use that in both Nav and SubNav. The benefit is that in the future if you ever need to make a change to these components in the future, the code is centralized in one component.